datanode/sqlstore/migrations/0076_teams_stats.sql

```diff
@@ -1,71 +1,83 @@
 -- +goose Up
 CREATE TABLE IF NOT EXISTS teams_stats
 (
-  team_id              BYTEA             NOT NULL,
-  party_id             BYTEA             NOT NULL,
-  at_epoch             BIGINT            NOT NULL,
-  total_quantum_volume HUGEINT           NOT NULL,
-  total_quantum_reward NUMERIC(1000, 16) NOT NULL,
-  games_played         JSONB             NOT NULL,
-  PRIMARY KEY (team_id, party_id, at_epoch)
+    team_id              BYTEA             NOT NULL,
```

Comment:
You can't update a migration that has already been deployed

Reply:
Indeed, that's a mistake.